service/src/main/java/uk/gov/hmcts/reform/fpl/config/utils/FixedListMappingType.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.fpl.config.utils;
+
+public enum FixedListMappingType {
```

Comment:
I expected to have one enum per fixed list type. I do not like having one big bag for all enumerated values. Please split it into the following enums:
- OrderType
- EmergencyProtectionOrderType
- EmergencyProtectionOrderDirectionType

After that create 3 separate pebble filters as well.